webapp/store/content/publishers/kde.yaml

```diff
@@ -564,8 +564,8 @@ snaps: [
         "origin": "kde",
         "package_name": "rocs",
         "publisher": "KDE",
-        "summary": "Graph Theory Tool for Professors and Students.",
```

Comment:
Find/Replace all ? ;)

Reply:
Weird! Must have been! Will put that back in :D